pipeline_reviews/extract.py

```diff
@@ -44,43 +42,39 @@ def get_reviews_for_game(game_id: int, cursor: str) -> dict:
         review_dict["last_timestamp"] = datetime.fromtimestamp(
             review["timestamp_updated"]).strftime("%Y-%m-%d %H:%M:%S")
         review_dict["playtime_at_review"] = review["author"]["playtime_at_review"]
-        review_dict["full_playtime"] = review["author"]["playtime_forever"]
         page_reviews.append(review_dict)
     return {"next_cursor": next_cursor, "reviews": page_reviews}
 
 
 def get_all_reviews(game_ids: list[int]) -> None:
     """Combines all reviews together and all review
     information together to be set in a data-frame"""
-    reviews_info = []
     all_reviews = []
 
     for game in game_ids:
-        reviews_info.append(get_review_info_for_game(game))
-        if reviews_info[-1]["number_of_total_reviews"] == 0:
-            continue
-
-        cursor_list = ["*"]
-
-        for page in range(int(reviews_info[-1]["number_of_total_reviews"]/100)+2):
-            api_response = get_reviews_for_game(game, cursor_list[page])
-            if "error" in list(api_response.keys()):
-                continue
-            cursor = api_response["next_cursor"]
-            page_reviews = api_response["reviews"]
-            if not page_reviews or cursor in cursor_list:
-                break
-            if not cursor in cursor_list:
-                cursor_list.append(cursor)
-            all_reviews.extend(page_reviews)
-    make_csv_files(all_reviews, reviews_info)
-
-
-def make_csv_files(all_reviews: list[dict], reviews_info: list[dict]) -> None:
+        number_of_total_reviews = get_review_info_for_game(game)
+
+        if number_of_total_reviews:
+            cursor_list = ["*"]
+
+            for page in range(int(number_of_total_reviews/100)+2):
+                api_response = get_reviews_for_game(game, cursor_list[page])
+
+                if not "error" in list(api_response.keys()):
+                    cursor = api_response["next_cursor"]
+                    page_reviews = api_response["reviews"]
+                    if not page_reviews or cursor in cursor_list:
+                        break
+                    if not cursor in cursor_list:
+                        cursor_list.append(cursor)
+                    all_reviews.extend(page_reviews)
+    make_csv_files(all_reviews)
```

Comment:
Just to note - this was changed mainly due to taking out the continue statements as they're classed as unprofessional